src/main/java/org/jabref/gui/entryeditor/EntryEditor.css

```diff
@@ -122,3 +122,45 @@
 .bibtexSourceCodeArea .text {
     -fx-fill: -fx-text-background-color;
 }
+
+.latex-citations-tab {
+    -fx-background-color: -fx-control-inner-background;
+    -fx-padding: 20px;
+}
+
+.latex-citations-title-text {
```

Comment:
I guess some of these css definitions are the same as the one in the dialog. If this is indeed the case, then please move them to the base.css file to remove the code duplication.

Reply:
Sure. I have created specific style classes just in case we decide to redesign the displaying. If it is OK, I will reduce duplications.